.github/workflows/RequestReview.yml

```diff
@@ -0,0 +1,102 @@
+name: Request review
+
+on:
+  pull_request:
+    types: ["review_requested"]
+
+jobs:
+    assign_and_move_card:
+        name: Assign issue to reviewer and move Kanban card
+        runs-on: ubuntu-latest
+
+        steps:
+            # https://github.com/actions/github-script
+            - uses: actions/github-script@v3
+              with:
+                    github-token: ${{secrets.GITHUB_TOKEN}}
+                    script: |
+                        const toDoColumn = 4971951;
+                        const inProgressColumn = 4971952;
+                        const reviewInProgressColumn = 4971953;
+                        const reviewApprovedColumn = 4971954;
+                        const doneColumn = 4971955;
+                        //
+                        async function getIssue(issue_number) {
+                            try {
+                                return (await github.issues.get({
+                                    owner: context.repo.owner,
+                                    repo: context.repo.repo,
+                                    issue_number
+                                })).data;
+                            }
+                            catch{
+                                console.log("Issue not found: #" + issue_number);
+                                return null;
+                            }
+                        }
+                        //
+                        async function findCard(content_url) {
+                            // Columns are searched from the most probable one
+                            const allColumns = [reviewInProgressColumn, reviewApprovedColumn, inProgressColumn, toDoColumn, doneColumn];
+                            for (let i = 0; i < allColumns.length; i++) {
+                                let cards = await github.projects.listCards({ column_id: allColumns[i] });
+                                let card = cards.data.find(x => x.content_url == content_url);
+                                if (card) {
+                                    return card;
+                                }
+                            }
+                            console.log("Card not found for: " + content_url);
+                            return null;
+                        }
+                        //
+                        async function processIssue(issue) {
```

Comment:
Shouldn't we check first if the issue is open before making any changes?

Reply:
If we do a mistake by `Fixes #<closed issue>` than I would still move it on Kanban to raise our attention to it. 
I've added the check for direct PR processing, because there's no reason to do it there.